source/extensions/common/wasm/wasm.cc

```diff
@@ -231,6 +231,9 @@ void Wasm::getFunctions() {
 #define _GET(_fn) wasm_vm_->getFunction("envoy_" #_fn, &_fn##_);
   _GET(on_resolve_dns)
 #undef _GET
+#define _GET(_fn) wasm_vm_->getFunction("" #_fn, &_fn##_);
```

Comment:
If this is defined here it should have the envoy_ prefix until we move it to proxy-wasm, then it will get a proxy_ prefix.

Reply:
sounds good, just fixed